src/game_config_manager.cpp

```diff
@@ -211,8 +211,7 @@ void game_config_manager::load_game_config(bool reload_everything, const game_cl
 			game_config::config_cache_transaction main_transaction;
 			
```

Comment:
Remember that there's still whitespace to fix on line 212.

Reply:
Updated.